Tensile/Source/client/include/ResultReporter.hpp

```diff
@@ -124,6 +124,9 @@ namespace Tensile
             const std::string DeviceIndex         = "device-idx";
             const std::string FanSpeedRPMs        = "fan-rpm";
             const std::string HardwareSampleCount = "hardware-samples";
+            const std::string GfxFrequency = "gfx-frequency(median)"; // GPU clock in Mhz 
+            const std::string Power =  "power(median)"; // power consumption by the GPU
+            const std::string TemperatureHot = "hotspot-temperature(median)"; // GPU temperature             
```

Comment:
Please follow formatting of the variables above for consistency

Reply:
Existing metrics just collect the average information alone.  vector size is 1 for all the values . just accumulates all the values in the same index.
New metrics we are planning to log the detailed report like min, max, average, median and all individual data points during kernel execution hence slightly different. slightly deviating from existing way for minimal overhead. 

yes formatting needs changes.